drivers/storage/template/tests/DRIVERNAME_test.go

```diff
@@ -0,0 +1,44 @@
+// +build !libstorage_storage_driver libstorage_storage_driver_DRIVERNAME
+
+package DRIVERNAME
+
+import (
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/codedellemc/libstorage/api/server"
+	"github.com/codedellemc/libstorage/api/types"
+)
+
+// Put contents of sample config.yml here
+var (
+	configYAML = []byte(``)
+)
+
+var volumeName string
```

Comment:
I'd remove the volumeName and volumeName2 as they're just going to confuse people.

> Are we supposed to use those? What are those for?

People can be very literal.